i18n/de.json

```diff
@@ -9,20 +9,21 @@
   "UIKit.CreatableSelectInput.noOptionsMessageWithoutInputValue": "",
   "UIKit.DateInput.labelRange": "bis",
   "UIKit.DateTimeInput.labelRange": "bis",
-  "UIKit.LocalizedMultilineTextInput.collapse": "",
-  "UIKit.LocalizedMultilineTextInput.expand": "",
-  "UIKit.LocalizedMultilineTextInput.hide": "",
-  "UIKit.LocalizedMultilineTextInput.missingRequiredField": "",
-  "UIKit.LocalizedMultilineTextInput.show": "",
-  "UIKit.LocalizedTextInput.collapse": "",
-  "UIKit.LocalizedTextInput.expand": "",
-  "UIKit.LocalizedTextInput.missingRequiredField": "",
+  "UIKit.LocalizedMultilineTextInput.collapse": "Verbergen",
+  "UIKit.LocalizedMultilineTextInput.expand": "Anzeigen",
+  "UIKit.LocalizedMultilineTextInput.hide": "Sprachen verbergen",
+  "UIKit.LocalizedMultilineTextInput.missingRequiredField": "Pflichtfeld*, Geben Sie mindestens einen Wert an",
+  "UIKit.LocalizedMultilineTextInput.show": "Alle Sprachen anzeigen",
+  "UIKit.LocalizedTextInput.collapse": "Sprachen verbergen",
+  "UIKit.LocalizedTextInput.expand": "Alle Sprachen anzeigen",
+  "UIKit.LocalizedTextInput.missingRequiredField": "Pflichtfeld*, Geben Sie mindestens einen Wert an",
   "UIKit.MoneyInput.chevronLabel": "Dropdown öffnen/schliessen",
   "UIKit.MultilineTextInput.collapse": "Ausblenden",
   "UIKit.MultilineTextInput.expand": "Erweitern",
   "UIKit.SelectInput.noOptionsMessageWithInputValue": "",
   "UIKit.SelectInput.noOptionsMessageWithoutInputValue": "",
+  "UIKit.TextField.missingRequiredField": "",
```

Comment:
What about this one?

I guess nobody is using it yet, so it's not a problem. We might soon reuse our error message, so it won't be necessary to repeat the common ones for every field. I'm fine with leaving it empty.

Reply:
I could just make it the same as `UIKit.LocalizedTextInput.missingRequiredField`? 🤷‍♂️